tests/test_translate.py

```diff
@@ -275,9 +293,12 @@ def test_parallel_savepoint(
     failure_stride,
     subtests,
     caplog,
+    python_regression,
     xy_indices=False,
 ):
     caplog.set_level(logging.DEBUG, logger="fv3core")
+    if python_regression and not testobj.python_regression:
+        pytest.xfail(f"python_regression not set for test {test_name}")
```

Comment:
I see this only in the parallel translate class, which mirrors the code here. Can regression tests be serial?

Reply:
For this PR they can only be parallel. But a future PR could extend this for serial tests if we decide to go that route. The motivation would be somewhat different from the purpose of this PR.